var/spack/repos/builtin/packages/predixy/package.py

```diff
@@ -15,6 +15,7 @@ class Predixy(MakefilePackage):
 
     license("BSD-3-Clause")
```

Comment:
v7.0.0: "This a Predixy FreeEdition, It's not open source now" Limits on number of clients.

Reply:
@wdconinc do you have a preference on whether we should keep this in or put in a note about the change to closed source instead? I'm not sure I understand if their new license allows for redistribution.

Reply:
I have no preference. I am not a user of this package :-)

Upon further inspection, they still list BSD as LICENSE in the repo. That still allows redistribution. I think adding the version is fine. The only thing to consider is whether it makes sense to make the default a hobbled version. Since it is still BSD, we should even be allowed to distribute the patches to un-hobble it...